guides/app-guides/borrow-ageur.md

```diff
@@ -0,0 +1,79 @@
+---
+description: How to borrow agEUR from the Angle App
+---
+
+# Borrowing agEUR 
```

Comment:
Need to include wherever is fit details about permit (different types of permit here), wrapping and approval transactions